src/live-region/internal.tsx

```diff
@@ -106,7 +105,14 @@ export default React.forwardRef(function InternalLiveRegion(
   }));
 
   return (
-    <TagName ref={mergedRef} {...baseProps} className={clsx(styles.root, className)} hidden={hidden}>
+    <TagName
+      ref={mergedRef}
+      {...baseProps}
+      className={clsx(styles.root, testUtilStyles.root, className)}
+      hidden={hidden}
+      // https://github.com/facebook/react/issues/17157#issuecomment-2127180687
+      {...{ inert: hidden ? '' : undefined }}
```

Comment:
you had a discussion on that topic, what is the outcome? We're keeping it as is?